python/dgl/nn/pytorch/conv/dotgatconv.py

```diff
@@ -41,6 +41,8 @@ class DotGatConv(nn.Module):
         same value.
     out_feats : int
         Output feature size; i.e, the number of dimensions of :math:`h_i^{(l+1)}`.
+    num_heads : int
+        Number of head in Multi-Head Attention
```

Comment:
Please also change the formula in the doc and verify which dimension the multi-head will applied on.

Reply:
OK